README.md

```diff
@@ -32,6 +32,16 @@
 - Start the server with `docker-compose up` and wait for the Docker images to start up.
 - Ideally, the server will boot up and start on its own, but this doesn't happen all the time. If it looks like the startup process has stopped, go to IP you got from `docker-machine ip` with your web browser. The server should then spin up and start serving the site.
 
+### Setup for Mac (El Caption)
```

Comment:
I'll have to reject all these steps altogether. It basically mirrors the general setup listed above, but adds `docker-compose build`. instead, please add that missing step on the main "Setup" section.

NOTE: There is no need to get `brew` or `mysql` installed onto your computer, since Docker is handling all package managing as well as MySQL.

Reply:
It complained that I didn't have  Mysql installed when I tried it. 

Ok, I'll add it there.

Reply:
I would assume that as a Rails dev your first intent was to run `bundle exec rails server`, in which case, yup, it would complain about MySQL. However, with the Docker workflow, you do `docker-compose up` to run all the stuff. In addition, you can run common Rails commands by "sshing" into the container first (`docker-compose run rails bash`), then running any Rails command (e.g. `rails console`).